documentation/asciidoc/topics/proc_disabling_encryption.adoc

```diff
@@ -0,0 +1,18 @@
+[id='disabling_encryption-{context}']
+= Disabling Encryption
+You can disable encryption in test and development environments so clients do not need TLS certificates to establish connections with {brandname}.
+For production environments, you should always use encryption to secure connections and restrict access to data.
```

Comment:
I think this is a bit strong. We've added the ability to disable TLS specifically because people do want it in production envrionments. More specifically for when clients utilising the endpoints reside in the same k8s cluster as the Infinispan server(s).

Reply:
I think it might be worth adding a note saying something like:

"We do not recommend disabling encryption in production enviroments where the Infinispan endpoints are exposed outside of the k8s cluster via `spec.Expose.Type`"

Reply:
@ryanemerson thanks, dogg. updated and squashed.